### PR TITLE
Scorecard page status page tooltips

### PR DESCRIPTION
**Pull request by randytpierce**
I'd like to have this reviewed to make sure my tooltips (main.js) are actual english and meteorology speak. Also I believe that the fontawesome package only needs to be in the app package.json not in MATScommon. I'dd like to see if that is true. If it is, Molly might want to include this in development soon.

### CODE REVIEW

**Comment by ian-noaa** [comment]
A thought - it looks like `.../apps/scorecard/.meteor/versions` still has fontawesome in it. https://github.com/NOAA-GSL/MATS/blob/72fa2049b679596f2fda87c77b0f492d9bc0401a/apps/scorecard/.meteor/versions#L36

Maybe that's causing meteor to still pull the old version of fontawesome in? I wonder what would happen if we removed the `.meteor/versions` file and let meteor recreate it.

File: apps/scorecard/server/main.js
Comment (mollybsmith-noaa) [inline apps/scorecard/server/main.js:47]: This is a duplicate line. `'scorecard-ends-on'` is already the fourth line in this `hideOtherFor`

File: apps/scorecard/server/main.js
Comment (mollybsmith-noaa) [inline apps/scorecard/server/main.js:55]: I think in the future we should explore ways for these messages to be in common code, so we don't have to copy and paste them into each app.

File: apps/scorecard/server/main.js
Comment (mollybsmith-noaa) [inline apps/scorecard/server/main.js:249]: Please revert this to "block". The hideOtherFor will take care of it. The reason it isn't working is because there was a duplicate line, as noted earlier.

**Review by mollybsmith-noaa — CHANGES_REQUESTED** [review]
I'm a little concerned that the line `<div id="{{name}}-item" class="item btn-group"  role="group" style="display:{{display}}"  data-toggle="tooltip" data-placement="{{tooltipPlacement}}" title="{{tooltipTitle}}"` is going to throw errors in the other apps. Have you tried it there?

File: apps/scorecard/server/main.js
Comment (randytpierce) [reply apps/scorecard/server/main.js:55]: Good catch, We should do that the way internationalization is often done, i.e. put a ne type in matstypes.tooltips.whatnot that contains each seperate and common message.

File: apps/scorecard/server/main.js
Comment (randytpierce) [reply apps/scorecard/server/main.js:249]: good catch

**Review by mollybsmith-noaa — APPROVED** [review]
At this point my comments are addressed, but I'd still like to do the release before this is merged.

**Comment by randytpierce** [comment]
> A thought - it looks like `.../apps/scorecard/.meteor/versions` still has fontawesome in it. https://github.com/NOAA-GSL/MATS/blob/72fa2049b679596f2fda87c77b0f492d9bc0401a/apps/scorecard/.meteor/versions#L36

Maybe that's causing meteor to still pull the old version of fontawesome in? I wonder what would happen if we removed the `.meteor/versions` file and let meteor recreate it. 

.meteor/versions is taken care of automatically when you do a meteor reset. I hadn't done a meteor reset. I didn't think we were controlling .meteor.

**Comment by randytpierce** [comment]
We are going to wait until after the release to merge this into dev and I am going to apply the necessary changes for fontawesome-free to all of the apps today and push them to this branch as well.

File: apps/scorecard/server/main.js
Comment (randytpierce) [reply apps/scorecard/server/main.js:53]: I can do this today and push that change to this branch as well since we are waiting for the merge. This is a good idea.

**Comment by ian-noaa** [comment]
Sounds good. Ideally, I think the fontawesome work would be tracked in its own issue/branch since this PR is nicely scoped to the tooltip work you've been doing. Up to you though. 

And good to know that fontawesome is needed both places. I could see that making sense so you don't have dependencies leaking between your various libraries and your app. So that's actually a bit of a relief. :sweat_smile: 

Meteor's not explicit about it in their docs but it seems like everything in `.meteor` (except `.meteor/local`) is intended to be version controlled. The directory contents (`.meteor/packages` and `.meteor/versions`) appear to be analogous to npm's `package.json` & `package-lock.json`, respectively.

**Comment by randytpierce** [comment]
I have added fontawesome into all the apps now, and I have cleaned some things up. Let's give this pull request another go, please.

**Comment by mollybsmith-noaa** [comment]
I approve this pull request to be merged into development.

**Pull request merged**
